soccer_vision_3d_rviz_markers/conversion.py

```diff
@@ -104,7 +104,7 @@ def obstacle_to_marker(msg: Obstacle) -> Marker:
     return marker
 
 
-def robot_to_markers(msg: Robot) -> list[Marker]:
+def robot_to_markers(msg: Robot) -> Marker:
```

Comment:
This function returns a list of Markers. But the previous notation was wrong as `list` should be capitalized and imported. 

Add this import: `from typing import List`
And change the type to `List[Marker]`